connectors/sources/confluence.py

```diff
@@ -959,7 +959,7 @@ async def search_by_query(self, query):
                 "_id": entity_details.get("id"),
                 "title": entity.get("title"),
                 "_timestamp": entity.get("lastModified"),
-                "body": entity.get("excerpt"),
+                "body": entity_details.get("body", {}).get("storage", {}).get("value"),
```

Comment:
Can we use the same safe function as [here](https://github.com/elastic/connectors/pull/2598)?

Reply:
Sure, i'll raise a follow-up PR and address this in all relevant places once [that PR](https://github.com/elastic/connectors/pull/2598) is merged.